fenix/app/src/androidTest/java/org/mozilla/fenix/experimentintegration/launchSimScript.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e
+
+echo "Waiting emulator is ready..."
+~/Library/Android/sdk/emulator/emulator -avd Pixel_3_API_28 -wipe-data -no-boot-anim -screen no-touch &
```

Comment:
Nit: Not sure if we should use a newer emulator.
When doing this we are assuming the emulator is installed, wondering if there is a way to install and launch an emulator, if not we need to be sure the emulator is available where this tests are going to run.

Reply:
This file should be removed actually. I am going to use the config in the ```syncintegration``` folder, that still uses the same Pixel 3 though.